src/fish_tests.cpp

```diff
@@ -4328,6 +4329,11 @@ void history_tests_t::test_history_races_pound_on_history(size_t item_count, siz
 }
 
 void history_tests_t::test_history_races() {
+    // This always fails under WSL
```

Comment:
Pretty sure this one typically works for me, WSL 1, provided I do it on the linux filesystem. Fails a ton on OpenBSD, tho.

On the windows filesystem the uvar signalling fails?

Reply:
On my WSL machine, the fish source code is in a shared NTFS directory but are you quite sure you're actually running the `history_races` test under WSL? This commit was just re-adding the patch that was dropped in the cleanup of `fish_tests.cpp` in an earlier commit in this branch; I actually (silently) patched out `history_races` under WSL (v1, at least) years ago so there's a good chance you're not actually running it under WSL at all.

Reply:
Ah, okay. That's it - it doesn't run the history_races test. I missed that because it doesn't say it's been skipped. It just doesn't mention it at all.